src/pasttle/views/index.html

```diff
@@ -11,7 +11,17 @@
       </div><!--end of section-->
       <div class="section separator">
         <h2>CLI Examples:</h2>
-        <div>
+        <script>
+        function show_examples() {
+            document.querySelector('#cli_examples').style.display = "block"
+            document.querySelector('#cli_helpers').style.display = "block"
+            document.querySelector('#cli_examples_holder').style.display = "none"
+        }
+        </script>
+        <div id="cli_examples_holder" onclick="show_examples()" style="cursor: pointer">
```

Comment:
This should also be underlined to show it's actually a link (I know it has the hand pointer style, but still.)